### ISSUE TITLE
setup / alter docker-compose file for development purposes (debezium, kafka, redis, elasticsearch, mongo, etc etc)

### ISSUE THREAD

**Issue by P-T-I** [cve-search/CveXplore#217]

**Comment by P-T-I** [comment]
@baonq-me  Are you willing to take-up this issue?

**Comment by baonq-me** [comment]
Yes, please assign it to me. I already did it in my internal system.

**Comment by P-T-I** [comment]
Awesome! I hope to make a start with the sql models (#214) somewhere today!

**Comment by P-T-I** [comment]
@baonq-me #247 has just merged into master; it holds the first setup of the database models; needs a full test, but the first setup is there! As part of the tests I've created a development folder with a docker-compose file I'm using to test the migrations into a mysql database; it might be something you can build / extend on.
